app/app.go

```diff
@@ -584,6 +573,7 @@ func NewExocoreApp(
 			app.StakingKeeper.EpochsHooks(), // at this point, the order is irrelevant.
 			app.ExomintKeeper.EpochsHooks(), // however, this may change once we have distribution
 			app.AVSManagerKeeper.EpochsHooks(),
+			app.OperatorKeeper.EpochsHooks(),
```

Comment:
EpochHooks wrapped the pointer of OperatorKeeper, and app.OperatorKeeper is initialized after this register with `NewOperatorKeerper`, so here in this hooks is will have a `nil` operatorKeeper

Reply:
fixed